with-native-tabs/README.md

```diff
@@ -0,0 +1,50 @@
+# Welcome to your Expo app 👋
```

Comment:
Follow template of other router pkgs like with-router-ai, especially the launch button.